src/app/helptext/task-calendar/replication.ts

```diff
@@ -0,0 +1,155 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T('Replication task name'),
+
+    direction_placeholder: T('Direction'),
+    direction_tooltip: T('Direct the flow of the ZFS snapshots. Choose between pushing\
+ snapshots to a remote host or pulling snapshots from a remote host.'),
+
+    transport_placeholder: T('Transport'),
+    transport_tooltip: T('Method for snapshot transfer:<ul>\
+ <li><i>SSH</i> is supported by most systems. It requires a previously created\
+ <a href="%%docurl%%/system.html%%webversion%%#ssh-connection" target="_blank">SSH connection</a>.</li>\
+ <li><i>SSH+NETCAT</i> uses SSH to establish a connection to the remote system, then uses\
+ an unencrypted data stream to improve data transfer speeds. This is only an option when\
+ replicating to a FreeBSD system that has <a href="https://github.com/freenas/py-libzfs"\
+ target="_blank">py-libzfs</a> installed.</li>\
+ <li><i>LOCAL</i> replicates snapshots to another dataset within the system.</li>\
+ <li><i>LEGACY</i> uses the replication engine implemented in FreeNAS 11.2 and earlier.</li></ul>'),
+
+    ssh_credentials_placeholder: T('SSH Connection'),
+    ssh_credentials_tooltip: T('Configured SSH connection to use for the replication.\
+ Choose from a list of connections configured in System > <a href="%%docurl%%/system.html%%webversion%%#ssh-connection"\
+ target="_blank">SSH connection</a>.'),
+
+    netcat_active_side_placeholder: T('Netcat Active Side'),
+    netcat_active_side_tooltip: T('System that will open ports for the direct connection'),
```

Comment:
Not clear what this means.

Reply:
Which system will open ports for high-speed replication direct connection: local or remote?

Reply:
Not clear how "active" means "opens ports".  Not even clear whether "opens ports" means opening ports in a firewall or opening receiving ports.

Reply:
@themylogin ping?

Reply:
@wblock okay, let's name it differently, any way you think is right, but the meaning is that the side of connection chosen there will open TCP port, and other side should be able to connect to it.